djangocms_versioning/admin.py

```diff
@@ -135,6 +141,176 @@ def has_change_permission(self, request, obj=None):
         return super().has_change_permission(request, obj)
 
 
+class ExtendedVersionAdminMixin(VersioningAdminMixin):
+    """
+    Extended VersionAdminMixin for common/generic versioning admin items
+    """
+    list_display_links = None
+
+    class Media:
+        js = ("admin/js/jquery.init.js", "djangocms_versioning/js/actions.js")
+        css = {
+            "all": ("djangocms_versioning/css/actions.css",)
+        }
+
+    def get_version(self, obj):
+        """
+        Return the latest version of a given object
+        :param obj: Versioned Content instance
+        :return: Latest Version linked with content instance
+        """
+        return obj.versions.all()[0]
+
+    def get_versioning_state(self, obj):
+        """
+        Return the state of a given version
+        """
+        return self.get_version(obj).get_state_display()
+
+    get_versioning_state.short_description = _("State")
+
+    def get_author(self, obj):
+        """
+        Return the author who created a version
+        :param obj: Versioned content model Instance
+        :return: Author
+        """
+        return self.get_version(obj).created_by
+
+    get_author.short_description = _("Author")
+
+    def get_modified_date(self, obj):
+        """
+        Get the last modified date of a version
+        :param obj: Versioned content model Instance
+        :return: Modified Date
+        """
+        return self.get_version(obj).modified
+
+    get_modified_date.short_description = _("Modified")
+
+    def get_list_display_configuration(self):
+        """
+        Return the configured list display from cms config versionable object, if it is not configured,
+        return the default django value if list_display is not set.
+        """
+        versionable = versionables.for_content(self.model)
+        return versionable.admin_list_display_fields or ["__str__"]
+
+    def get_preview_method_configuration(self):
```

Comment:
```suggestion
    def _get_preview_url(self, obj):
```

This is getting a url not returning a function / class method.